units/systemd-sysupdated.service.in

```diff
@@ -0,0 +1,31 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=System Update Service
+Documentation=man:systemd-sysupdated.service(8)
+Documentation=man:org.freedesktop.sysupdate1(5)
+ConditionVirtualization=!container
```

Comment:
why that conditionalization?

Reply:
Containers are generally updated by the host system, and I'd expect that they don't have the privileges to tamper with their own backing storage in the ways sysupdate requires

Reply:
sure, the containers themselves are. but some other resource inside them might be something we could update. I don't think this should really be forbidden entirely.

Reply:
please drpo this conditionalization